From 47766883e6366cce5684a36eb9975a91db8b157c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9bastien=20Wilmet?= <swilmet@gnome.org>
Date: Sun, 7 Jun 2015 13:42:26 +0200
Subject: [PATCH 08/26] spell: don't #include gedit-metadata-manager.h (private
 header)

The gedit_metadata_manager_* functions are anyway not used in the spell
plugin.
---
 plugins/spell/gedit-spell-plugin.c | 1 -
 1 file changed, 1 deletion(-)

--- a/plugins/spell/gedit-spell-plugin.c
+++ b/plugins/spell/gedit-spell-plugin.c
@@ -42,7 +42,6 @@
 #include "gedit-automatic-spell-checker.h"
 
 #ifdef G_OS_WIN32
-#include <gedit/gedit-metadata-manager.h>
 #define GEDIT_METADATA_ATTRIBUTE_SPELL_LANGUAGE "spell-language"
 #define GEDIT_METADATA_ATTRIBUTE_SPELL_ENABLED  "spell-enabled"
 #else
